xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
can you make it branched? it is hard to tell what is modified.

Reply:
What is "branched" ? 

I agree it is hard - git knows the file has moved but can't show the diff in this UI

Reply:
still waiting for the "branched" explanation..

Reply:
For some reason I cannot comment on your other comment: at the time of first checkin git knows the file has been moved as it shows this:
```
git status
On branch xds-server-test
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	renamed:    xds/src/test/java/io/grpc/xds/internal/sds/XdsSdsClientServerTest.java -> xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java
...
```
But if you do a git log for that commit it shows delete and add of 2 different files:
```
xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java                          | 224 +++++++++++++++++++++++++++++++++++++++++++++
xds/src/test/java/io/grpc/xds/internal/sds/XdsSdsClientServerTest.java             | 193 --------------------------------------
```
Unless you have an idea how to fix it, I am going to resolve this comment

Reply:
don't worry about it. i reviewed the file anyways.